packages/adminrouter/extra/src/README.md

```diff
@@ -479,6 +479,13 @@ as `.js` and `.css` files. This is due to the fact that the linked resources wil
 be reachable only in their relative location
 `<dcos-cluster>/services/<application ID><link>`.
 
+### Disabling URL path rewriting for selected applications
+Some applications like Jenkins or Artifactory, permit specifying a context path (e.g. `/service/jenkins`) that is used while generating application URLs (both relative and absolute). However, normally Adminrouter strips these, and this can lead to breakages, esp. for assets like JavaScript files and images.
+
+It is possible to workaround it by using another Nginx in the application container which rewrites URLs from / to /service/jenkins. This isn't ideal though, so an extra Marathon label is recognized by Admin Router. If a task has `DCOS_SERVICE_REWRITE_REQUEST_URLS` label set to `no` or `0` or `false`, Admin Router will not strip the context path and the upstream request will be made with the same the URL path as the client request has been made.
```

Comment:
> set to `no` or `0` or `false`

Is this a design choice made for this PR or is it something common in DC/OS / AR?

Reply:
> Is this a design choice made for this PR or is it something common in DC/OS / AR?
 
If it is a design choice, I think maybe `true, false` could be the only allowed values, else you get an error. This prevents things like `False` and so on causing confusion.

Reply:
Let's agree on `false` (string) and `false` (boolean) as @jgehrcke pointed out . 

I was trying to follow Robustness Principle here, but maybe you are right and we should be a bit more strict. Wrt. `true` I am not so sure about it - if the label is absent, then the default is `true`. The label should only be present if there is a need to not to rewrite the upstream URL and it should assume one of two values: `false` (str) or `false` (bool). Having explicit `true` would increase the complexity of the patch without any real gain.

What do you think?

Reply:
> Having explicit true would increase the complexity of the patch without any real gain.

ack!